src/app/controllers/Cake.class.ts

```diff
@@ -0,0 +1,56 @@
+import { CakeInterface, statusCake } from './../models/cake.model';
```

Comment:
I recommend renaming this file to `cake.class.ts` using lower case. In TypeScript, we follow the JavaScript style guides: https://google.github.io/styleguide/jsguide.html#file-name

Reply:
Perfect! Thanks :)